crates/solidity/src/solc/solc_compiler.rs

```diff
@@ -52,46 +63,73 @@ impl Compiler for SolcCompiler {
         allow_paths: Option<String>,
     ) -> anyhow::Result<StandardJsonOutput> {
         let version = self.version()?;
+        trace!("In standard_json");
```

Comment:
Please remove all logging, it is unrelated to the bugfix this PR is addressing.

If you want to enable logging or tracing please do it in another PR, use the `log` crate and initialize the `env_logger` in the main function. make sure to log in all relevant places and use `